### ISSUE TITLE
Open, run and extract results from an existing xml file with RCS

### ISSUE THREAD

**Issue by MHarrouni** [idea-statica/ideastatica-public#390]
I have written a python code which creates an xml for IDEA. Opening it by hand and running works perfectly fine. However, I haven’t managed to open it, run and extract results from it using coding. Trying to run it I encountered the following issue:

"
Traceback (most recent call last):

  File C:\ProgramData\anaconda3\Lib\site-packages\spyder_kernels\py3compat.py:356 in compat_exec
    exec(code, globals, locals)

  File c:\workspace.git\entrace\write plaxis to idea-4.py:1219
    rcsController.OpenIdeaProject(r"C:\Workspace.git\Entrace\IDEA input\Results_Full_Model_Vak2_s7_toIDEA.xml")

Exception: Item not found in file 'C:\Workspace.git\Entrace\IDEA input\Results_Full_Model_Vak2_s7_toIDEA.xml'.
   at CI.ArchiveTools.ProjectExtractor.GetPasswordFromArch(String archFileName)
   at CI.ArchiveTools.ProjectExtractor.Extract(String archFileName, String directory)
   at CI.Concrete.SingleCheckProject.RcsProject.UnCompressProject(String fileName)
   at CI.Concrete.SingleCheckProject.RcsProject.CreateProject(String fileName)
   at IdeaStatiCa.RcsController.IdeaRcsController.OpenIdeaProject(String ideaProjectPath)
"

Maybe the code is meant solely to run .rcs-files. Maybe the .xml-file isn't providing the necessary info. Or maybe the code is just wrong. 

Is it possible to look into this issue?

The xml-file and the code used to run the file are in the attached zip-file. 

[IDEA Issue.zip](https://github.com/idea-statica/ideastatica-public/files/13557488/IDEA.Issue.zip)

**Comment by nathanlukeIDEA** [comment]
Hi @MHarrouni,

Yes, this method opens an existing Rcs project.

If you want to Create and RCS project from IOM.xml then you should use this method:

```
string xmlfile = "C:\\Users\\NathanLuke\\Desktop\\rcsTest\\RcsOpenModel2.xml";
string filemsg = "C:\\Users\\NathanLuke\\Desktop\\rcsTest\\RcsOpenModel2msg.txt";

rcsController.OpenIdeaProjectFromIdeaOpenModel(xmlfile, "Column", filemsg); 
```

**Comment by MHarrouni** [comment]
In the python console the message "False" is returned, when trying to run the method mentioned above. Please note that the xml-file is not written using the IOM-module but written line by line using python. 
![image](https://github.com/idea-statica/ideastatica-public/assets/152961319/3e1df359-58fd-4de9-9c8d-2b7a87ad5d0e)

![image](https://github.com/idea-statica/ideastatica-public/assets/152961319/f0caed3c-cfe1-42da-b014-d5bf759079e5)

Also, what is the purpose of the txt-file?

**Comment by nathanlukeIDEA** [comment]
The .txt file outputs issues encountered on the creation of the rcs project. If you open it may tell you why you received a false output from the method.

**Comment by nathanlukeIDEA** [comment]
Hi @MHarrouni, I hope you have fixed your issues here. Due to problems we were encountering with the older RCS Api we have now just released a newer version with v23.1.3 of IDEA StatiCa. The new version has a dedicated python client which you can use and will hopefully solve a lot of issues. 

https://developer.ideastatica.com/docs/api/rcs/rcs_api_getting_started.html?tabs=python
